src/Console/Mappings/Traits/UpdatesAlias.php

```diff
@@ -47,7 +47,7 @@ protected function getActiveIndex(string $alias):string
      */
     protected function getAlias(string $mapping):string
     {
-        return preg_replace('/[0-9_]+/', '', $mapping, 1);
+        return preg_replace('/^\d{4}\_\d{2}\_\d{2}\_\d{6}\_/', '', $mapping, 1);
```

Comment:
Very minor thing, but I don't think you need backslashes before the underscores as underscores aren't considered a special character.